app/collections/Classrooms.coffee

```diff
@@ -0,0 +1,17 @@
+Classroom = require 'models/Classroom'
+CocoCollection = require 'collections/CocoCollection'
+
+module.exports = class Classrooms extends CocoCollection
+  model: Classroom
+  url: '/db/classroom'
+  
+  initialize: ->
+    @on 'sync', =>
+      for classroom in @models
+        classroom.capitalizeLanguageName()
```

Comment:
Does it make more sense to have Classrooms know how to capitalize their own names on sync instead of the collection reaching in to do that for them?